CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.10.1-rc1
```

Comment:
This also should **not** be changed during the cherry-pick/rebase. Please do not make **ANY** changes in any of the cherry-picked commits.